.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.0.1
+current_version = 3.1.0
```

Comment:
Removing a variable is a backwards-incompatible change, so let's make this a major version bump, to `4.0.0`